syft/math.py

```diff
@@ -1008,3 +1008,43 @@ def split(tensor, split_size, axis=0):
     list_ = np.array_split(tensor.data, split_according, axis)
 
     return list(map(lambda x: TensorBase(x), list_))
+
+
+def cross(input, other, dim=-1):
```

Comment:
could you add an encryption [check](https://github.com/OpenMined/PySyft/blob/34af4f778c2b2f1a16a5a8aa505c37b7d9b18009/syft/math.py#L330) for the inputs that are passed?

Reply:
Have made the requested changes and updated the PR.